lib/tasks/review_profile.rake

```diff
@@ -1,5 +1,5 @@
 require 'io/console'
-
+# Note: Any modifications made here should be updated accordingly in the lib/action_account.rb file.
```

Comment:
Thanks for adding this note! Maybe even add a note like "this file should be going away soon!"